lib/components/steps/mapping/MappingSearch.tsx

```diff
@@ -3,30 +3,53 @@ import {Box, Button, InputAdornment, TextField} from "@mui/material";
 import {FilterIcon, SearchIcon} from "../../../icons";
 import Filters from "../../common/Filters.tsx";
 import { useDebounce } from "../../../hooks.ts";
-
+import { EntityType, CheckedState } from "../../../models.ts";
 
 interface MappingSearchProps {
-    onChange: (searchTerm: string) => void;
+    onChange: (searchTerm: string, checked: CheckedState) => void;
 }
 
 export default function MappingSearch({onChange}: MappingSearchProps) {
 
     const [searchString, setSearchString] = useState('');
+    const [checked, setChecked] = React.useState({
```

Comment:
suggestion (non-blocking): Rename 'checked' to something more informative like for example filtersState